frigate/app.py

```diff
@@ -584,19 +597,26 @@ def start_timeline_processor(self) -> None:
         )
         self.timeline_processor.start()
 
+    def start_embeddings_processor(self) -> None:
+        self.embeddings_processor = EmbeddingProcessor(
```

Comment:
This class will return and kill the thread immediately if `semantic_search` is not enabled. See: https://github.com/blakeblackshear/frigate/pull/8998/files#diff-bb5fcf4020ca2bf73e181a7b467529ec9f574c7292f720b742014f795b82054aR47-R48

Reply:
Makes sense, I figured that was the case but think it might be better to have it explicit in this file vs the implicit behavior. Not a big deal though